audinterface/core/feature.py

```diff
@@ -77,6 +77,18 @@ class Feature:
             Afterwards processing is applied to each segment
         keep_nat: if the end of segment is set to ``NaT`` do not replace
             with file duration in the result
+        min_signal_length: minimum signal length
```

Comment:
Should we call it `min_signal_dur` and `max_signal_dur` for consistency with `win_dur` and `hop_dur`?

Reply:
I switched to `min_signal_dur` and `max_signal_dur`